scala-thoughts.md

```diff
@@ -0,0 +1,29 @@
+
+The feeling of using Scala is slightly odd. Most things really well, but only because it
+"was made that way." Of course, that's the whole point of having a scalable language.
+You can feel domain-specific in a much larger range of places. Even so, it still feels
+a bit unsettling. Does it ever happen that accidental implicit casts cause huge issues, for example?
```

Comment:
I agree in that implicit types, especially for lambdas can be confusing and create more problems for the user than other systems would.